Luke_Smith_Portfolio.css

```diff
@@ -0,0 +1,126 @@
+body {aaa
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topnav {
+  overflow: auto;
+  background-color: #ffffff;
+}
+
+.topnav a {
+  float: left;
+  color: #000000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  width: auto;
+}
+
+.topnav a:hover {
+  background-color: #dddddd;
+  color: #000000;
+}
+
+body {
+  background: #555555;
+}
+
+/*About Me Section*/
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: #000000;
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.column {
+  float: left;
+  width: 33.3%;
+  margin-bottom: 16px;
+  padding: 0 8px;
+}
+
+.about-section {
+  padding: 50px;
+  text-align: center;
+}
+
+.container {
+  padding: 0 16px;
+}
+
+.container::after, .row::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+.title {
+  color: #000000;
+  font-style: italic;
+}
+
+.button {
+  border: none;
+  outline: 0;
+  display: inline-block;
+  padding: 8px;
+  color: #FFFFFF;
+  background-color: #000;
+  text-align: center;
+  cursor: pointer;
+  width: 100%;
+}
+
+.button:hover {
+  background-color: #555;
+}
+
+.maydenlogo {
+	width: 500px;
+}
+
+.maydenlogo img {
+	width: 100%;
+}
+
+/*Contact section links*/
+.contact:link, .contact:visited {
+  background-color: #FFFFFF;
+  color: #000000;
+  border: 2px #000000;
+  padding: 10px 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+ }
+
+.contact:hover, .contact:active {
+  background-color: #000000;
+  color: #FFFFFF;
+ }
+ 
+ /*Media queries*/
+ @media only screen and (max-width: 500px) {
+ 	body {
+ 	  padding: none;
+ 	  width: 100%;
+   }
+   
+  img {
+ 	 width: 90%;
+ 	 height: auto;
+  }
+  
+  .topnav a {
+ 	 font-size: 12px;
+	 padding-left: 0;
+	 padding-right: 0;	
+  }
+  
+  body {
+	  width: 90%;
+  }
+ }
```

Comment:
Tested the media query on the standard devices in Chrome's inspector, and it seems to be working as intended.